pymatviz/utils.py

```diff
@@ -699,8 +699,14 @@ def normalize_to_dict(
     Raises:
         TypeError: If the input format is invalid.
     """
+    if cls is None:
+        from pymatgen.core import Structure  # costly import
+
+        cls = Structure
+
     if isinstance(inputs, cls):
```

Comment:
Looks like we might need a temporary helper function to test Structure until all those changes go to pymatgen (would certainly take some time)